shared/nav.desktop.js

```diff
@@ -140,6 +145,20 @@ class Nav extends Component<void, Props, State> {
     return this.props.tabbedRouter.get('activeTab')
   }
 
+  _handleKeyDown (e: SyntheticKeyboardEvent) {
+    const modKey = process.platform === 'darwin' ? e.metaKey : e.ctrlKey
```

Comment:
Unclear we want this in production. Also we can use the named keys and not the keycode